aiodocker/images.py

```diff
@@ -14,6 +14,10 @@
     overload,
 )
 
+
+if TYPE_CHECKING:
+    from _typeshed import SupportsRead
```

Comment:
Sorry, an import from a private module is prohibited.
Please copy-paste `SupportsRead` protocol implementation here.